cms/core/blocks/section_blocks.py

```diff
@@ -41,15 +43,27 @@ class Meta:
         block_counts: ClassVar[dict[str, dict]] = {"related_links": {"max_num": 1}}
 
 
+class GlossarySectionBlock(StructBlock):
+    title = HeadingBlock(default="Definitions")
+    content = ListBlock(SnippetChooserBlock(GlossaryTerm))
```

Comment:
note: I was cross-referencing with Amelia's notes and it is possible we may want to move away from a dedicated section to just a "definitions" `ListBlock` within the main SectionContentBlock

See the "The ability to add a line of text below the definitions (within the content section) to link off to a more detailed glossary if needed." note on the ticket

Reply:
Yup, I had a chat with Mebin before I started to work on this ticket and decided to go with this. Personally I think the approach you've described here is more natural as it removes the extra step of selecting between Section and Glossary Section.

Two things to note if we decide to go with this aporoach:
- the behaviour of defaulting the section title to "Definitions" would have to be revisited - do we scrap this completely? or do we implement some custom logic to automatically fill it in if the definitions `ListBlock` is detected? (I'd be leaning towards removing it completely as it might be confusing for the users)
- the tests would have to be rewritten - particularly in functional tests we would have to remove the step where the user selects between Section and Glossary Section

Reply:
Great, thanks a lot!

Reply:
Just had another look at our messages aand I have to say my bad. What confused me is that you said that it should be part of `SectionStoryBlock`, but technically it should be part of `SectionContentBlock` (which holds all the blocks available to be selected in the content of the section). I'll change that ASAP!

Reply:
> Just had another look at our messages aand I have to say my bad. What confused me is that you said that it should be part of `SectionStoryBlock`, but technically it should be part of `SectionContentBlock` (which holds all the blocks available to be selected in the content of the section). I'll change that ASAP!

Ah right, no worries, I should have been clearer. For now, I would hold fire, Kacper, until we get a reply from Amelia. Given it has been done and if that is their preference, we may as well keep it as is.

Reply:
@MebinAbraham fwiw you are cc-ed in my email, so should get the reply too.